html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label><br />
+        <input type="email" /><br />
+      </div>
+
+      <div>
+        <label for="radio-button">Why is coding cool</label><br />
+        <select id="is-coding-cool" name="simple">
+          <br />
+          <option selected>Select an option</option>
+          <option>Becuase it&#39;s fun</option>
+          <option>Because it&#39;s interesting</option>
+          <option>Because it&#39;s powerful</option></select
```

Comment:
This options have no values! The first one makes sense because it is "Select and option" but the other options should have values or else we aren't submitting anything with our form!